src/core/client/admin/components/UserHistoryDrawer/UserHistoryDrawerAllComments.tsx

```diff
@@ -40,7 +40,7 @@ const UserHistoryDrawerAllComments: FunctionComponent<Props> = ({
   }, [loadMore]);
 
   const hasMore = relay.hasMore();
-  const comments = user ? user.allComments.edges.map(edge => edge.node) : [];
+  const comments = user ? user.allComments.edges.map((edge) => edge.node) : [];
```

Comment:
Won't really have an impact either

Reply:
If you banned the user, or suspended them, wouldn't it trigger a re-render because the user prop did change? Does the relay fragment container prevent re-renders if the fragment contents do not change?

Reply:
Yes it shouldn't rerender because this component doesn't ask for the user banned status.